### PR TITLE
MOL-441: remove customer creation feature

### PR DESCRIPTION
**Pull request by boxblinkracer**
remove customer feature for now

config is set to always create NO customer
and config in admin has been removed for the user

### CODE REVIEW

**Review by barbieswimcrew — APPROVED** [review]

**Comment by schmitzcarsten** [comment]
should already be solved in v1.4.0. Please rebase and check

File: src/Resources/config/config.xml [hunk 1]
```diff
@@ -73,12 +73,6 @@
             <label lang="de-DE">Standard Shopware Verhalten bei fehlgeschlagenen Bestellungen benutzen</label>
             <label lang="nl-NL">Gebruik standaard Shopware redirect voor mislukte bestellingen.</label>
         </input-field>
-        <input-field type="bool">
```
Comment (schmitzcarsten) [inline src/Resources/config/config.xml:76]: pls rebase. This should already be integrated

**Comment by schmitzcarsten** [comment]
already integrated

**Comment by schmitzcarsten** [comment]
already integrated

**Pull request closed without merging**